### PR TITLE
PSR-407 Added transfers in CYA page

### PR DESCRIPTION
**Pull request by westwater**

### CODE REVIEW

**Review by narendrapaduchuri — DISMISSED** [review]

File: app/controllers/nonsipp/employercontributions/EmployerContributionsCYAController.scala [hunk 1]
```diff
@@ -298,7 +298,7 @@ object EmployerContributionsCYAController {
             // append "any contributions by other employers" row to final section
           ) :?+ Option.when(journeyIndex + 1 == employerCYAs.length)(
             CheckYourAnswersRowViewModel(
-              Message("employerContributionsCYA.row.contributionOtherEmployer", membersName),
+              Message("transfersInCYAController.rows.reportAnotherTransfer", membersName),
```
[the remark this answers is not in the document]
Comment (westwater) [reply app/controllers/nonsipp/employercontributions/EmployerContributionsCYAController.scala:301]: Woops my mistake

File: app/navigation/nonsipp/ReceiveTransferNavigator.scala [hunk 2]
```diff
@@ -79,24 +82,70 @@ object ReceiveTransferNavigator extends JourneyNavigator {
...
+          controllers.nonsipp.receivetransfer.routes.TransfersInCYAController
+            .onPageLoad(srn, index, NormalMode)
+
+        case WhenWasTransferReceivedPage(srn, index, secondaryIndex) =>
+          controllers.nonsipp.receivetransfer.routes.TransfersInCYAController
+            .onPageLoad(srn, index, NormalMode)
+
+        case DidTransferIncludeAssetPage(srn, index, secondaryIndex) =>
+          controllers.nonsipp.receivetransfer.routes.ReportAnotherTransferInController
+            .onPageLoad(srn, index, secondaryIndex, CheckMode)
```
[the remark this answers is not in the document]
Comment (westwater) [reply app/navigation/nonsipp/ReceiveTransferNavigator.scala:130]: Yeah, we're not navigating back to the CYA page for this case so left it in check mode

File: app/pages/nonsipp/receivetransfer/package.scala [hunk 3]
```diff
@@ -26,4 +29,13 @@ package object receivetransfer {
     val memberTransfersIn: JsPath = memberDetails \ "memberTransfersIn"
   }
 
+  def transferInPages(srn: Srn, index: Max300, secondaryIndex: Max5): List[QuestionPage[_]] = List(
+    DidTransferIncludeAssetPage(srn, index, secondaryIndex),
```
[the remark this answers is not in the document]
Comment (westwater) [reply app/pages/nonsipp/receivetransfer/package.scala:33]: Why do we do that? surely we'd want to remove all pages for the journey (both indexes)?

File: app/controllers/nonsipp/employercontributions/EmployerContributionsCYAController.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (kberec) [reply app/controllers/nonsipp/employercontributions/EmployerContributionsCYAController.scala]: Sorry, I should have said. One more occurrence few lines down.

File: app/pages/nonsipp/receivetransfer/package.scala (hunk 3 above)
[the remark this answers is not in the document]
Comment (kberec) [reply app/pages/nonsipp/receivetransfer/package.scala:33]: I vaguely remember that we needed to redirect to this page and it was expected the previous answer to be pre-selected on the screen. 
But we can leave here as is, and come back to it if we find out there is a need to do that.

**Review by kberec — APPROVED** [review]

**Pull request merged**